pkgs/by-name/bo/bombsquad/package.nix

```diff
@@ -0,0 +1,63 @@
+{
+  stdenv,
+  lib,
+  python311Full,
+  SDL2,
+  mesa,
+  libGL,
+  openal,
+  libvorbis,
+  fetchurl,
+  makeWrapper,
+  patchelf
+}:
+stdenv.mkDerivation rec {
+  version = "1.7.33";
+  name = "bombsquad-${version}";
```

Comment:
We no longer use `name` over decades. Use pname instead

Reply:
I'll fix that but... Why???